tests/unit/SidebarFilterTest.js

```diff
@@ -744,54 +655,6 @@ describe('Sidebar', () => {
         });
 
         describe('in GSD (focus) mode', () => {
-            it('is shown when it is unread', () => {
```

Comment:
I think this test is redundant. We have a test above that checks to see if the archived report is visible in focus mode.